cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -239,3 +239,4 @@ spec:
       {{- if .Values.affinity }}
       affinity: {{ toYaml .Values.affinity | nindent 8 }}
       {{- end }}
+      dnsPolicy: {{ .Values.dnsPolicy }}
```

Comment:
Maybe I'd prefer leaving the default "" 🤔
```suggestion
      {{- with .Values.affinity }}
      dnsPolicy: {{ . }}
      {{- end }}
```

Reply:
Thank you for the feedback. I changed it assuming you meant `.Values.dnsPolicy` and not `.Values.affinity`.

Reply:
Aha, yeah 😅 you got it right